Description: Remove webfonts from asciidocor html pages.
 The asciidocor html pages use webfonts by default, this disables the
 webfonts when generating html pages with asciidoctor.
Author: Jaimos Skriletz <jaimosskriletz@gmail.com>
Forwarded: https://github.com/fvwmorg/fvwm3/pull/932
Last-Update: 2023-12-02
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -58,6 +58,6 @@ $(html_MANS): $(SECTION_FILES)
 	"$(ASCIIDOC)" -b manpage -a "$(patsubst %.1,%,$@)" "$<" -o "$@"
 
 %.html: %.adoc
-	"$(ASCIIDOC)" -b html5 -a toc -a "$(patsubst %.html,%,$@)" "$<" -o "$@"
+	"$(ASCIIDOC)" -b html5 -a toc -a webfonts! -a "$(patsubst %.html,%,$@)" "$<" -o "$@"
 
 endif
